src/string_util.h

```diff
@@ -1,10 +1,11 @@
 #ifndef BENCHMARK_STRING_UTIL_H_
 #define BENCHMARK_STRING_UTIL_H_
 
-#include <sstream>
-#include <string>
-#include <utility>
-#include "internal_macros.h"
```

Comment:
I'm pretty sure `internal_macros.h` is required to configure correctly.

Reply:
Yeah, i was just looking into that